packages/blobdex/src/index.js

```diff
@@ -0,0 +1,81 @@
+import { base58btc } from 'multiformats/bases/base58'
+export * from './types.js'
+
+export class Blobdex {
+  #items
+
+  constructor() {
+    this.#items = new Map()
+  }
+
+  /**
+   * len numbers the count of items in this Blobdex.
+   *
+   * @returns {number}
+   */
+  len() {
+    return this.#items.size
+  }
```

Comment:
FYI you could expose this as a property accessor:

```suggestion
  get len() {
    return this.#items.size
  }
```

Usage:

```js
const idx = new Blobdex()
console.log(idx.len)
```

Maybe consider using `size` since this is more like a set...